src/components/social-share/social-share.scss

```
@@ -0,0 +1,7 @@
+.__social-share-container {
+  display: flex;
+  align-items: center;
+  & > * {
+    padding: 4px !important;
```

We can use `margin` and avoid using `!important`
```suggestion
    margin: 4px;
```